### PR TITLE
Baby steps: Adding support for RequestMapping in Spring

### PR DESCRIPTION
**Pull request by phillip-kruger**
`@RequestMapping` working on GET as an alternative to `@GetMapping`
Basic `@PostMapping` and alternative `@RequestMapping` working

@ebullient @MikeEdgar - let me know what you think

### CODE REVIEW

File: extension-spring/src/main/java/io/smallrye/openapi/spring/ParameterProcessor.java [hunk 1]
```diff
@@ -1036,6 +1034,11 @@ static String pathOf(AnnotationTarget target) {
                     path = methodAnnotation;
                 }
             }
+            // Also support @RequestMapping
+            AnnotationInstance methodAnnotation = target.asMethod().annotation(SpringConstants.REQUEST_MAPPING);
+            if (methodAnnotation != null && methodAnnotation.value() != null) {
+                path = methodAnnotation;
+            }
```
Comment (MikeEdgar) [inline extension-spring/src/main/java/io/smallrye/openapi/spring/ParameterProcessor.java:1041]: Should this code also apply to classes with the `@RequestMapping` annotation?

Comment (ebullient) [reply extension-spring/src/main/java/io/smallrye/openapi/spring/ParameterProcessor.java:1041]: Y. @RequestMapping is a valid class-level annotation. Not sure that needs to be on this pass..

Comment (phillip-kruger) [reply extension-spring/src/main/java/io/smallrye/openapi/spring/ParameterProcessor.java:1041]: Yes maybe. We should create a test case for that then ...

**Comment by phillip-kruger** [comment]
Merging this one, will create new tests for the class level case.

**Pull request merged**